cypress/support/commands.js

```diff
@@ -23,3 +21,26 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    const isDev = Cypress.env('NODE_ENV') === 'dev';
+    const baseUrl = isDev ? Cypress.config('baseUrl') : `${Cypress.config('baseUrl')}/dashboard`;
+    cy.visit(`${baseUrl}/auth/login`);
+    cy.intercept('GET', '/v3-public/authProviders').as('authProviders');
+    cy.wait('@authProviders').then(res => {
+      const { CSRF } = cookie.parse(document.cookie);
+      cy.request({
+        method: 'POST',
+        url: '/v3-public/localProviders/local?action=login',
+        body: {
+          "description":"UI session",
+          "responseType":"cookie",
+          "username":"admin",
+          "password":"password1234"
```

Comment:
Should this be pointing at the env username/passwords instead of being static?

Reply:
Sorry, I forgot to change it back when I tested it.

Reply:
Done